_data/kvjoshi/kvjoshi.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Karmavir Joshi
+institution: Marwadi Education Foundation Group of Institutions
+profile_pic: kv.jpg
```

Comment:
Hi @kvjoshi, this image name does not match the image you have uploaded. Please make it so they both have the same name.

Reply:
fixed it